Change theme color to blue

Josh-Cena opened the pull request.

Josh-Cena committed:
Change theme color to blue

yechs reviewed (dismissed):
我倒是觉得首页那个 了解更多 的 button，可以把字变成白色的
黑色的字和背景色的 contrast 不是很足够

其它颜色上面的话都 ok 啦，那个蓝色稍稍明亮一点点会更好（个人的 taste）
现在也已经挺不错的啦

Josh-Cena committed:
Brighten theme color

Josh-Cena committed:
Invert get started button color

Josh-Cena commented:
按钮颜色我不会改orz

现在是用 `!important` 覆盖的，不知道有没有优雅一点的方法

yechs committed:
Avoid !important in CSS

Achived by selecting the anchor using a unique id as CSS id selectors
take precedence over class selectors

yechs reviewed (approved):
> 现在是用 !important 覆盖的，不知道有没有优雅一点的方法

解决了，其它的看上去都挺好的
